### PR TITLE
Update for clarity

### PR DESCRIPTION
**Pull request by TheMitch2**
Update clear ops wording to clarify what is being cleared and restored.

### CODE REVIEW

**Comment by TheMitch2** [comment]
Would it be appropriate to merge this into 0.19 at some point?

**Comment by cloudkucooland** [comment]
Is this still a work-in-progress or is it ready to merge?

**Comment by le-jeu** [comment]
the pr has to be clean first (package-lock)

File: src/code/translations/english.json [hunk 1]
```diff
@@ -237,7 +237,7 @@
   "OP_BUTTON": "Operation",
   "OP_SETTINGS_BUTTON": "Op ⚙",
   "OP_SETTINGS_TITLE": "Operation Settings",
-  "OPTION": "option",
```
Comment (le-jeu) [inline src/code/translations/english.json:240]: in fact this string looks never used

File: src/code/translations/english.json [hunk 2]
```diff
@@ -171,7 +171,7 @@
   "MADRID_WAS_TAKEN": "Madrid Protocol",
   "MADRID_TITLE": "Madrid Protocol",
   "MADRID": "Draw",
-  "MADRID_NOTSET": "baselink not set.",
+  "MADRID_NOTSET": "Baselink not set.",
```
Comment (le-jeu) [inline src/code/translations/english.json:174]: this string looks unused

**Comment by TheMitch2** [comment]
Removed a few unused strings. There are probably more, but this will take time to find.

**Comment by TheMitch2** [comment]
At long last this is ready for review.

**Review by cloudkucooland — APPROVED** [review]
looks good (English).

File: src/code/translations/english.json (hunk 1 above)
Comment (cloudkucooland) [reply src/code/translations/english.json:241]: My understanding is a pass to look for unused keys is coming soon.

**Comment by TheMitch2** [comment]
There are a few things that still need doing. As for clarity in English, I feel this is good. Finding all the unused keys and getting all keys into all languages is looking to be a much larger project.

**Review by le-jeu — APPROVED** [review]

**Pull request merged**